### DISCUSSION TITLE
Network/rendering fix issues and bugs (unstable)

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15024] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

-flashlights are weird and flickery (while mounted to guns at least, not sure if while held)
-doors can be shot through sometimes, if recently closed (?) (enough to kill someone)
-platforms and stairs sometimes teleport people up multiple times, resulting in fall damage
-getting stuck behind doors after walking through is still common 

![watch video](https://github.com/user-attachments/assets/e0ee1d6d-638d-42e3-b97d-6ebd0c6a7f1b)

-stairs still can't be a custom length please please please please please please please please 
-harpoons loaded in harpoon guns render wrong ![image](https://github.com/user-attachments/assets/e5f0d4b8-4ab4-41fc-823c-6003ded9a24a)

-depth charges and railgun shells are rendered sideways 
![image](https://github.com/user-attachments/assets/65598c49-81f9-4006-bba9-8540a19afab9)
![image](https://github.com/user-attachments/assets/bff2344b-7018-465b-b337-d541ab2d144d)

-players climbing ladders sometimes looks like having a grand-mal seizure (super jittery)

### Reproduction steps

ougghjjjjjjjjjjjjjj play the game

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
All of these should've been addressed in the last few unstable builds, closing.
